CHANGELOG.yaml

```
@@ -1,3 +1,10 @@
+3.6.6:
+  date: 2020-09-04
+  fixed bugs:
+    - >-
+      Fixed a issue where in case of an invalid auth type update() was setting 
+      auth to default from the existing type.
```

Seems a little confusing, how about something along the lines of this:

```suggestion
      Fixed an issue in RequestAuth where type was set to previous auth type when update() called with invalid type.
```